src/main/java/com/cascadebot/cascadebot/commands/moderation/UnbanCommand.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.moderation;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandMain;
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.exceptions.InsufficientPermissionException;
+
+public class UnbanCommand implements ICommandMain {
+
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length == 0) {
+            context.replyDanger("Not enough arguments (No specified member)");
+            return;
+        }
+        String targetUser = context.getMessage(0);
+        try {
+            context.getGuild().getController().unban(targetUser).queue();
+            context.replyInfo("User <@" + targetUser + "> has been Unbanned!");
```

Comment:
Why the <@ > ? Just put the name here :)

Reply:
You also need to put the message inside the queue success consumer and add an error message